src/Vlingo.Xoom.Streams.Tests/Operator/FlatMapperTest.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using Vlingo.Xoom.Streams.Source;
+using Xunit;
+
+namespace Vlingo.Xoom.Streams.Tests.Operator
+{
+  public class FlatMapperTest : IDisposable
+  {
+    [Fact]
+    public void TestThatPropagatesRecordsFormTheProvidedSource()
+    {
+      // GIVEN
```

Comment:
Not really an issue, just was wondering if `Given/When/Then` comments are necessary? Is that how it was in Java implementation?

Reply:
Yeah, not really, and not how implemented in Java, was just for readability and split responsibility when starting by implementing tests.